scripts/gen-ceph-kustomize.sh

```diff
@@ -205,6 +217,46 @@ function create_volume {
     oc rsh -n $NAMESPACE ceph ceph fs volume create cephfs >/dev/null || true
 }
 
+function config_ceph {
+    # Define any config option that should be set in the mgr database
+    # via associative arrays and inject to the Ceph Pod
+    # Define and set config options
+    echo "Apply Ceph config keys"
+    declare -A config_keys=(
+        ["rgw_keystone_url"]="http://keystone-public-openstack.$DOMAIN"
```

Comment:
As per the way endpoints are exposed, identity (keystone) will create a GenericRoute [1] based on the `instance.ServiceName`, `EndpointType` and `Namespace`.
I'm hardcoding the `Namespace` here as it's the default in both development and Prow jobs, otherwise it can be an additional parameter, which I can introduce in the next PS.

[1]  https://github.com/openstack-k8s-operators/lib-common/blob/modules/test/v0.1.1/modules/common/route/route.go#L53

Reply:
Done.